modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/asset/display/page/portlet/DepotEntryDisplayPageFriendlyURLResolver.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.depot.web.internal.asset.display.page.portlet;
+
+import com.liferay.asset.display.page.portlet.BaseAssetDisplayPageFriendlyURLResolver;
+import com.liferay.depot.model.DepotEntry;
+import com.liferay.depot.service.DepotEntryLocalService;
+import com.liferay.layout.display.page.LayoutDisplayPageProvider;
+import com.liferay.petra.string.CharPool;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.feature.flag.FeatureFlagManagerUtil;
+import com.liferay.portal.kernel.portlet.FriendlyURLResolver;
+import com.liferay.portal.kernel.portlet.constants.FriendlyURLResolverConstants;
+import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.service.ServiceContextThreadLocal;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Marco Leo
+ */
+@Component(service = FriendlyURLResolver.class)
+public class DepotEntryDisplayPageFriendlyURLResolver
+	extends BaseAssetDisplayPageFriendlyURLResolver {
+
+	@Override
+	public String getDefaultURLSeparator() {
+		return FriendlyURLResolverConstants.URL_SEPARATOR_ASSET_LIBRARY;
+	}
+
+	@Override
+	public String getKey() {
+		return DepotEntry.class.getName();
+	}
+
+	@Override
+	public boolean isURLSeparatorConfigurable() {
+		return true;
```

Comment:
It would be fine to hide the url separator configurable behind a FF and also add the key for the url separator label : com.liferay.depot.model.DepotEntry-url-separator configuration to the translations.